pyproject.toml

```diff
@@ -36,9 +36,11 @@ pyarrow = "^13"
 pie-modules = ">=0.11.1,<0.13.0"
 torch = [
     # default
-    {version = "^2.1.0+cpu", source = "pytorch", markers = "sys_platform != 'darwin' or platform_machine != 'x86_64'"},
+    {version = "^2.1.0+cpu", source = "pytorch", markers = "sys_platform != 'darwin' or platform_machine != 'arm64' and platform_machine != 'x86_64'"},
```

Comment:
what is the precedence fop the `and` and `or` operators? having both next to each other without any brackets looks quite ambiguous...

Reply:
I'll add brackets for the `and` operation.